instance/api/openedx_appserver.py

```diff
@@ -98,5 +98,5 @@ def make_active(self, request, pk):
             return Response(
                 {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
             )
-        app_server.instance.set_appserver_active(app_server.pk)
-        return Response({'status': 'App server updated.'})
+        set_appserver_active(app_server.pk)
+        return Response({'status': 'App server activation initiated.'})
```

Comment:
@smarnach I think a side effect of this would be to lose the ability to run playbooks from different version of ansible, right? Given how incompatible version of ansible can be, that might be useful to preserve - or at least, be conscious of the choice if we don't. 

Another possibility would be to only build each environment once, and only cleaning them up on restarts of the instance manager? Identifying them by venv name would allow to easily group identical environments, and have a default one, without losing the flexibility of specifying a specific venv on a command execution?